File: Semestrul 4/Medii de proiectare si programare/Laboratoare/FirmaTransportC#/Persistence/utils/PasswordUtils.cs
namespace FirmaTransportC_.Persistence.utils
{
    public static class PasswordUtils
    {
        public static string Hash(string plain)
        {
            return BCrypt.Net.BCrypt.HashPassword(plain);
        }

        public static bool Verify(string plain, string hashed)
        {
            return BCrypt.Net.BCrypt.Verify(plain, hashed);
        }
    }
}

File: Semestrul 3/Metode avansate de programare/Laboratoare/Laborator_GUI/Laborator_GUI/src/main/java/org/example/laboratorGUI/utils/PasswordUtils.java
package org.example.laboratorGUI.utils;
import org.mindrot.jbcrypt.BCrypt;

public class PasswordUtils {
    public static String hash(String plain) {
        return BCrypt.hashpw(plain, BCrypt.gensalt());
    }

    public static boolean verify(String plain, String hashed) {
        return BCrypt.checkpw(plain, hashed);
    }
}
